include/ck/tensor_operation/gpu/device/impl/device_contraction_multiple_d_xdl_cshuffle.hpp

```diff
@@ -146,7 +145,8 @@ template <index_t NumDimM,
           index_t CShuffleNXdlPerWavePerShuffle,
           typename CDEBlockTransferClusterLengths_MBlock_MPerBlock_NBlock_NPerBlock,
           index_t CDEBlockTransferScalarPerVector_NPerBlock,
-          LoopScheduler LoopSched = make_default_loop_scheduler()>
+          typename ComputeDataType = ADataType,
+          LoopScheduler LoopSched  = make_default_loop_scheduler()>
```

Comment:
Are we sure there is no one passing a loop scheduler explicitly right now? If there is such a case, it would break cause loop scheduler would be taken as a compute data type.

It seems like we don't do that anywhere in CK, but it's possible that hipTensor or any other library does that.